tsconfig.build.json

```diff
@@ -0,0 +1,4 @@
+{
+    "extends": "./tsconfig.json",
+    "include": ["./src"]
```

Comment:
Apologies, but I don't quite get this: which files were included so far? And why not just add the `include` statement to the original tsconfig?

Reply:
> which files were included so far

In the original `tsconfig`, I suppose, all files will be included. 
I believe that is fine since the original tsconfig is mainly used by VS Code / ide.

> why not just add the include statement to the original tsconfig?

Adding includes to the original tsconfig will prevent tsconfig from being applied to files outside of the `src` folder.

----

My intention here with `tsconfig.buid.json` `include` is to restrict rollup's typescript compiler `include` scope to files within `src` folder only